Ports/guile/package.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=guile
+version=3.0.8
+files="https://ftpmirror.gnu.org/gnu/guile/guile-${version}.tar.gz guile-${version}.tar.gz
+https://ftpmirror.gnu.org/gnu/guile/guile-${version}.tar.gz.sig guile-${version}.tar.gz.sig
+https://ftpmirror.gnu.org/gnu/gnu-keyring.gpg gnu-keyring.gpg"
```

Comment:
Instead of signatures, we tend to use SHA-256 now. Downloading the signature from the same source has no added benefit in terms of security, only integrity.